### PR TITLE
fix: Adding filter in academic term

### PR DESCRIPTION
**Pull request by anupamvs**

### CODE REVIEW

**Comment by anupamvs** [comment]
@mergifyio backport version-12-hotfix

File: erpnext/education/doctype/fee_structure/fee_structure.js [hunk 1]
```diff
@@ -9,6 +9,14 @@ frappe.ui.form.on('Fee Structure', {
 	},
 
 	onload: function(frm) {
+		frm.set_query("academic_term", function(){
```
Comment (deepeshgarg007) [inline erpnext/education/doctype/fee_structure/fee_structure.js]:
```suggestion
		frm.set_query("academic_term", function() {
```

File: erpnext/education/doctype/fee_structure/fee_structure.js [hunk 2]
```diff
@@ -9,6 +9,14 @@ frappe.ui.form.on('Fee Structure', {
...
+			return{
```
Comment (deepeshgarg007) [inline erpnext/education/doctype/fee_structure/fee_structure.js]:
```suggestion
			return {
```

File: erpnext/education/doctype/fee_structure/fee_structure.js [hunk 3]
```diff
@@ -9,6 +9,14 @@ frappe.ui.form.on('Fee Structure', {
...
+				"filters":{
```
Comment (deepeshgarg007) [inline erpnext/education/doctype/fee_structure/fee_structure.js]:
```suggestion
				"filters": {
```

File: erpnext/education/doctype/fee_structure/fee_structure.js [hunk 4]
```diff
@@ -9,6 +9,14 @@ frappe.ui.form.on('Fee Structure', {
...
+					"academic_year": (frm.doc.academic_year)
```
Comment (deepeshgarg007) [inline erpnext/education/doctype/fee_structure/fee_structure.js]:
```suggestion
					"academic_year":  frm.doc.academic_year
```

**Review by deepeshgarg007 — CHANGES_REQUESTED** [review]

**Review by deepeshgarg007 — APPROVED** [review]

**Pull request merged**